src/components/AppShell/AppShell.jsx

```diff
@@ -99,6 +112,7 @@ AppShell.propTypes = {
 };
 
 AppShell.defaultProps = {
+  featureFlips: ['enableReply'],
```

Comment:
The default value of this prop should be an empty array but I'm passing `enableReply` for now to not make this a breaking change.

This should be a temporal change and we should be able to change the default value soon.